src/snapred/backend/recipe/algorithm/LiteDataCreationAlgo.py

```diff
@@ -0,0 +1,99 @@
+import json
+
+from mantid.api import *
+from mantid.api import AlgorithmFactory, PythonAlgorithm, mtd
+from mantid.kernel import Direction
+
+from snapred.backend.recipe.algorithm.MantidSnapper import MantidSnapper
+
+name = "LiteDataCreationAlgo"
+
+
+class LiteDataCreationAlgo(PythonAlgorithm):
+    def PyInit(self):
+        self.declareProperty("InputWorkspace", defaultValue="", direction=Direction.Input)
+        self.declareProperty("OutputWorkspace", defaultValue="", direction=Direction.Output)
+        self.setRethrows(True)
+        self.mantidSnapper = MantidSnapper(self, name)
+
+    def PyExec(self):
+        self.log().notice("Lite Data Creation START!")
+        # define super pixel dimensions
+        superPixel = [8, 8]
```

Comment:
Do we really want this hard-coded?  Is there a way to either read this from the instrument definition, or to input this some other way?

Reply:
There are no signifiers for super pixel classification within the Instrument defintion.